qt5/platforminputcontext/qfcitxplatforminputcontext.cpp

```diff
@@ -47,10 +47,18 @@ void setFocusGroupForX11(const QByteArray &uuid) {
         return;
     }
 
-    if (!QX11Info::isPlatformX11()) {
+    if (QGuiApplication::platformName() != QLatin1String("xcb")) {
         return;
     }
-    auto connection = QX11Info::connection();
+
+    auto native = QGuiApplication::platformNativeInterface();
+    if (!native) {
+        return;
+    }
+
+    auto connection = reinterpret_cast<xcb_connection_t *>(
+        native->nativeResourceForIntegration(QByteArray("connection")));
+
```

Comment:
I suggest check connection is not null here just in case.

Reply:
fixed